images/release/Dockerfile

```diff
@@ -3,9 +3,9 @@
 #
 # The standard name for this image is openshift/sti-release
 #
-FROM openshift/origin-base
+FROM registry.access.redhat.com/ubi8/ubi
```

Comment:
registry.redhat.io?  (assuming you're only running this locally where you can expect to have proper creds).

otherwise this will break on you someday.

Reply:
went with the location published on Scott's announcement. I understand why that was done (no pull secret needed, lowers the barrier to entry).